sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/autorest.md

```diff
@@ -83,6 +83,113 @@ directive:
     $.properties.upperBoundaryList.items["x-nullable"] = true;
 ```
 
+``` yaml
```

Comment:
Nope, but I can create a new one. I think we'll never close it, though.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/17787

Reply:
Is the intention to communicate to the service about this changes and make sure they address them in their swagger?

Reply:
Yes. However, based on previous experiences in which we faced literally the same issue, this tends to be treated as a very low priority issue and ends up never being fixed (specially since we can workaround it on the client side). You can see we already had a set of similar transforms in MA for the same reason (service team already aware).